rh_solutions.rb

```diff
@@ -0,0 +1,39 @@
+def mirror(list)
+
+  index = list.length-1
+
+  (index + 1).times do
+    list << list[index]
+    index -= 1
+  end
+
+  return list
```

Comment:
This looks good! The only thing is that it modifies the original array and the instructions say to return a new array (and not modify the original)